src/constants/errorMessages.constant.ts

```diff
@@ -1,3 +1,5 @@
 export const statusMessages = {
-  USER_ALREADY_EXISTS: 'User with this email already exists'
+  USER_ALREADY_EXISTS: 'User with this email already exists',
+  INCORRECT_PASSWORD: 'Incorrect password',
+  USER_NOT_FOUND: 'User not found'
```

Comment:
Not found message can be reusable and dynamic. Even more, all of them can be dynamic.

Reply:
Do you mean like this? To clarify which user or id is incorrect. 
``` USER_NOT_FOUND: (userId: string) => `User with ID ${userId} not found`, 
USER_ALREADY_EXISTS: (email: string) => `User with email ${email} already exists`,
```

Reply:
It is also possible solution but I mean `(entity: string, something: string) => `${entity} with ${something} already exists` .Something like this.